selfdrive/ui/ui.cc

```diff
@@ -193,10 +194,15 @@ static void update_state(UIState *s) {
       }
     }
   }
-  if (Hardware::TICI() && sm.updated("roadCameraState")) {
+  if (sm.updated("roadCameraState")) {
     auto camera_state = sm["roadCameraState"].getRoadCameraState();
-    float gain = camera_state.getGainFrac() * (camera_state.getGlobalGain() > 100 ? 2.5 : 1.0) / 10.0;
-    scene.light_sensor = std::clamp<float>((1023.0 / 1757.0) * (1757.0 - camera_state.getIntegLines()) * (1.0 - gain), 0.0, 1023.0);
+    if (Hardware::EON()) {
+      float gain = camera_state.getGainFrac();
+      scene.light_sensor = std::clamp<float>((1023.0 / 5408.0) * (5408.0 - camera_state.getIntegLines()) * (1.0 - gain), 0.0, 1023.0);
```

Comment:
This formula (max_t - t) * (max_gain - gain) feels a bit weird?

Reply:
shouldn't it be
(max_t * max_gain) - (t * gain)
?

Reply:
I think of it as the proportion of EV

Reply:
But if the gain goes up, wouldn't that influence the meaning of the max time as well? The current output seems to be pretty linear when I go through the whole range of exposure.